sysid-application/src/main/native/include/sysid/Util.h

```diff
@@ -46,6 +46,8 @@ static constexpr const char* kUnits[] = {"Meters",  "Feet",      "Inches",
  */
 void CreateTooltip(const char* text);
 
+void CreateErrorPopup(bool& isError, std::string_view errorMessage);
+
```

Comment:
This should have failed the documentation check

Reply:
That's because doxygen isn't processing any files. The docs it generates are blank.

Reply:
https://github.com/wpilibsuite/sysid/pull/283 at least generates something, but it still isn't warning on undocumented functions.